Source/BindableFunctions/Misc.cs

```diff
@@ -141,6 +166,12 @@ public static void ForceCameraFollow()
         [BindableMethod(name = "Clear White Screen", category = "Misc")]
         public static void ClearWhiteScreen()
         {
+            //fix white screen 
+            string wakeControl = "Dream Return";
```

Comment:
Does it need to be a different variable. It's only used once

Reply:
You can get rid of it it was used more before I simplified it